openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -268,8 +268,8 @@ The following requirements apply to OpenID for Verifiable Presentations via redi
 
 The following requirements apply to OpenID for Verifiable Presentations via the W3C Digital Credentials API:
 
-* Wallet Invocation is done via the W3C Digital Credentials API or an equivalent platform API. Any other mechanism, including Custom URL schemes, MUST NOT be used.
-* The Response Mode MUST be `dc_api.jwt`.
+* The Wallet MUST support Wallet Invocation via the W3C Digital Credentials API or an equivalent platform API. The Verifier MUST use Wallet Invocation via the W3C Digital Credentials API or an equivalent platform API. The Verifier MUST NOT use any other mechanism, including Custom URL schemes.
```

Comment:
Does the last sentence imply that the VP over DC API and VP via Redirects are mutually exclusive?

Reply:
My thinking was that the MUST NOT limited the use custom URIs entirely for a verifier wishing to implement the DC API flow, which would effectively make redirect based flow with custom URIs not allowed.

However, since it's for the DC API specific flow, I think it's reasonable to say that these requirements only apply during the DC API flow. Verifiers are permitted to choose which flow they use for any presentation, so they aren't restricted in being able to implement both flows.

Reply:
Yeah that makes sense. My thought is also that the MUST NOT applies specifically when doing the DC API flow. I.e. you are not allowed to do a redirect which then somehow returns a DC response.
I guess this is more of a structural issue with the layout of this section since it is kind of unclear what the scope of it is.

Reply:
I also agree with Kristina's proposal. In the original PR text, I tried to just rephrase what was already there.

Reply:
@fkj could you please commit my proposal as an author of a PR please?